src/Compilers/Core/Portable/ReferenceManager/CommonReferenceManager.Binding.cs

```diff
@@ -17,14 +19,30 @@ internal partial class CommonReferenceManager<TCompilation, TAssemblySymbol>
         ///
         /// The first element (index==0) of the assemblies array represents the assembly being built.
         /// One can think about the rest of the items in assemblies array as assembly references given to the compiler to
-        /// 
         /// build executable for the assembly being built. 
-        /// 
         /// </summary>
         /// 
-        /// <param name="assemblies">
-        /// The set of <see cref="AssemblyData"/> objects describing assemblies, for which this method should
-        /// resolve references and find suitable AssemblySymbols.
+        /// <param name="explicitAssemblies">
+        /// An array of <see cref="AssemblyData"/> objects describing assemblies, for which this method should
+        /// resolve references and find suitable AssemblySymbols. The first slot contains the assembly being built.
+        /// </param>
+        /// <param name="resolverOpt">
+        /// Reference resolver used to look up missing assemblies.
+        /// </param>
+        /// <param name="importOptions">
+        /// Import options applied to implicitly resolved references.
+        /// </param>
+        /// <param name="allAssemblies">
+        /// Updated array <paramref name="explicitAssemblies"/> with resolved implicitly referenced assemblies appended.
+        /// </param>
+        /// <param name="implicitlyResolvedReferences">
+        /// Implicitly resolved references.
+        /// </param>
+        /// <param name="implicitlyResolvedReferenceMap">
+        /// Maps index to all references (explicit + implicit) to an index of a resolved assembly in all assemblies (explicit + implicit).
```

Comment:
This comment feels confusing, it looks like this map doesn't have elements corresponding to explicit references.